docs/velox-backend-limitations.md

```diff
@@ -6,7 +6,7 @@ nav_order: 5
 This document describes the limitations of velox backend by listing some known cases where exception will be thrown, gluten behaves incompatibly with spark, or certain plan's execution
 must fall back to vanilla spark, etc.
 
-### Override of Spark classes
+### Override of Spark classes (For Spark3.2 and Spark3.3)
```

Comment:
If not officially supported spark3.2/3.3 version is used,  `NoSuchMethodError` can be thrown at runtime, like the below issue reported. Could you help clarify it here? Thanks!
https://github.com/oap-project/gluten/issues/4514

Reply:
sure, linked this issue